frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -75,7 +75,7 @@ class DocumentList extends Component {
   constructor(props) {
     super(props)
 
-    this.state.selectedRows = {}
+    this.state = this.defaultLocalState()
```

Comment:
I still don't see the need for this to be a function. I'd personally define the default state as an object within the constructor, which is where people expect it, and use that as the argument for `setState()` whenever needed. It still allows us to have conditionals and it feels more consistent with `defaultProps`. This is what I mean:

```js
constructor(props) {
  super(props)

  this.state = this.defaultState = {
    foo: 'bar',
    bar: props.baz ? 'yep' : 'nope'
  }

  // Or, if you don't want to chain assignments:
  this.defaultState = {
    foo: 'bar',
    bar: props.baz ? 'yep' : 'nope'
  }

  this.state = this.defaultState
}

someLifecycleMethod() {
  this.setState(this.defaultState)
}
```

If you still want to make it a method, I'd call it `getDefaultLocalState()` as function names should have a verb.